### DISCUSSION TITLE
二月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#767] [通知]
## 🐛 Bug 修复

- 移除登录失败限制 @Richasy (#766)
- 布局性能优化 @Richasy (#765)

## 🧰 管理与维护

- 更换gRPC为托管实现 @hez2010 (#707)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2202.2.0'>二月更新 - 2</a>.</em>

**Comment by xyl23333** [comment]
第一

**Comment by cworld1** [comment]
真勤快

**Comment by Alpha-lab1** [comment]
建议修复下[干净阅读]在win11中导入书籍闪退的问题

**Comment by GITHHHUB** [comment]
win10播放视频能明显看到卡顿，动起来的画面一顿一顿的，播放相关的几个设置都尝试改了改还是没用，不知道为啥

**Reply by hez2010** [reply]
关掉NVIDIA显卡设置的G-Sync/FreeSync

**Reply by GITHHHUB** [reply]
不行啊，都没有gsync的选项，应该是压根不支持gsync来着
